deploy/olm/storageos/storageos.clusterserviceversion.yaml

```diff
@@ -386,6 +386,11 @@ spec:
           StorageOS pods.
         displayName: Tolerations
         path: tolerations
+      - description: Name of the Kubernetes distribution in use, e.g.
+          `openshift`.  This will be included in the product telemetry (if
+          enabled), to help focus development efforts.
+        displayName: Kubernetes Distribution Name
+        path: k8sDistro
```

Comment:
Should also add in `/deploy/olm/csv-rhel/storageos.clusterserviceversion.yaml`

Reply:
doh! missed one...